CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+master
+------
+
+* store statement's created and stored property as `\DateTime` instead of
+  `int` timestamp.
```

Comment:
I would reword this to something like "the created and stored properties of the internal mapping `Statement` class are now instances of PHP's `\DateTime` class instead of integers representing UNIX timestamps"